unity-client/Assets/Scripts/MainScripts/DCL/Controllers/QuestsController/QuestsController.cs

```diff
@@ -70,6 +75,8 @@ public void InitializeQuests(List<QuestModel> parsedQuests)
         /// <param name="progressedQuest"></param>
         public void UpdateQuestProgress(QuestModel progressedQuest)
         {
+            Debug.Log(JsonUtility.ToJson(progressedQuest));
```

Comment:
Maybe we should remove this log? or we want it in the prod version?

Reply:
No we dont, in fact there's another one on top of the `InitializeQuests` method.

Thanks! nice catch.